client/src/pages/DeckBuilder/DeckBuilder.tsx

```diff
@@ -101,13 +102,23 @@ export const DeckBuilder = (props: DeckBuilderProps) => {
     // console.log(cardId);
     const newDeck = [...deck].filter((id) => id !== cardId);
     setDeck(newDeck);
+    if (face === cardId)
+      setFace(null);
+  };
+
+  const onFaceClick = (cardId: number) => (e: React.MouseEvent) => {
+    if (face === cardId)
+      setFace(null);
+    else
+      setFace(cardId);
+    e.stopPropagation();
   };
 
   const saveDeck = () => {
     request({
       method: 'post',
       url: 'deckBuilder/saveDeck',
-      data: { name: deckName, deck },
+      data: { name: deckName, face: face, deck },
```

Comment:
can use shorthand for `face:face`
```suggestion
      data: { name: deckName, face, deck },
```